priv/eleveldb.schema

```diff
@@ -158,8 +158,8 @@
 %% changed at any time, but will not impact data on disk until
 %% next time a file requires compaction.
 {mapping, "leveldb.compression", "eleveldb.compression", [
-  {default, on},
-  {datatype, flag},
+  {default, snappy},
+  {datatype, {enum, [on, off, true, false, snappy, lz4]}},
```

Comment:
Actually not the same.  I failed to copy this line to multibackend.  The above line appears to work with all old values ... cuz I tested true and false.  Original cuttlefish expected on/off then generated true/false.